README.md

```diff
@@ -29,26 +29,103 @@ git clone https://github.com/sap/service-fabrik-boshrelease
 cd service-fabrik-boshrelease
 ```
 
+### Deploying Service-Fabrik
 
-### Deploy from Sources
+#### Creating and uploading release:
 
-If you need to modify the sources (beyond the example manifest):
+  ##### Deploying from Sources
+
+  If you need to modify the sources (beyond the example manifest):
+  ```shell
+  ./scripts/update
+  bosh create release --force && bosh upload release # repeat these commands on every change
+  ```
+
+  #### Deploying from Release
+
+  If you do not need to modify the sources (beyond the example manifest):
+  ```shell
+  bosh upload release $(ls -1rv releases/service-fabrik/service-fabrik-*.yml | head -1)
+  ```
+
+#### Uploading the cloud-config (for AWS):
+Then, we need to upload the cloud-config required for service-fabrik on bosh.
+For that, we need to update the vars-files for the cloud-config. 
+This can be done in 2 ways, providing the parameters through command line or editing the vars-file directly.
+
+The vars file to be edited is `cloud-config-aws-vars.yml`. It can be found in the `templates` directory.
+Once the vars file is filled with proper details, the cloud-config can be uploaded:
 ```shell
-./scripts/update
-bosh create release --force && bosh upload release # repeat these commands on every change
 cd templates
-./make_manifest warden
-bosh -n deploy
+bosh –e bosh upload-cloud-config --vars-store=cloud-config-aws-vars.yml cloud-config-aws.yml
 ```
 
-### Deploy from Release
+Alternatively, the parameters can be passed through the command line as below:
+```shell
+cd templates
+bosh –e bosh upload-cloud-config -v availability_zone1="value1" \
```

Comment:
This is not needed. Rather someone would change the file.

Reply:
ok.. will modify this